app/models/vedastore/ballot_style.rb

```diff
@@ -0,0 +1,13 @@
+class Vedastore::BallotStyle < ActiveRecord::Base
+  include XsdRailsFunctions
+  include Vedaspace::BallotStyle
+  
+  belongs_to :election
+  has_one :external_identifier_collection, :as=>:identifiable
+  has_many :ordered_contests
```

Comment:
What do you think about defining the belongs_to/has_one/has_many relationships in the VEDaSpace model? Then we could automatically create the Rails associations from that, and reuse it for any other tools built on top of VEDaSpace. For example VEDaspector could use that directly, instead of the introspecting the Rails associations.